api-schema/xdb.yaml

```diff
@@ -521,4 +583,23 @@ components:
       enum:
         - TERMINATE # default behaviour. ProcessStatus becomes TERMINATED
         - FAIL # ProcessStatus becomes FAILED
+    ProcessExecutionPublishToLocalQueueRequest:
```

Comment:
Maybe just `PublishToLocalQueueRequest`? I feel like that's clear enough as it has `local` in the name